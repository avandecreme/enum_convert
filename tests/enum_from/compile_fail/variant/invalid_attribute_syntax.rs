use enum_convert::EnumFrom;

enum Source {
    Unit,
}

#[derive(EnumFrom)]
#[enum_from(Source)]
enum Target {
    #[enum_from(Source::Unit::Extra)] // Invalid syntax - too many segments
    Unit,
}

fn main() {}
