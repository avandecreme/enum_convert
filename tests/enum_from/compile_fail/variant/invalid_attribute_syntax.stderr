error: Expected Enum or Enum::Variant
  --> tests/enum_from/compile_fail/variant/invalid_attribute_syntax.rs:10:17
   |
10 |     #[enum_from(Source::Unit::Extra)] // Invalid syntax - too many segments
   |                 ^^^^^^^^^^^^^^^^^^^
